src/pages/index.js

```diff
@@ -182,10 +279,18 @@ profileEditButton.addEventListener("click", () => {
 
 profileFormElement.addEventListener("submit", handleProfileFormSubmit);
 
+avatarFormElement.addEventListener("submit", handleAvatarFormSubmit);
+
 newPostButton.addEventListener("click", () => {
   openModal(newPostModal);
 });
 
+deleteForm.addEventListener("click", handleDeleteSubmit);
```

Comment:
`deleteForm.addEventListener("click", handleDeleteSubmit);
`

Please, don't **click** on the **form**. You should **submit** it